lib/zclient.c

```diff
@@ -4520,6 +4520,25 @@ static void zclient_event(enum zclient_event event, struct zclient *zclient)
 	}
 }
 
+enum zclient_send_status zclient_interface_set_arp(struct zclient *client,
+						   struct interface *ifp,
+						   bool arp_enable)
+{
+	struct stream *s;
+
+	s = client->obuf;
+	stream_reset(s);
+
+	zclient_create_header(s, ZEBRA_INTERFACE_SET_ARP, ifp->vrf->vrf_id);
+
+	stream_putl(s, ifp->vrf->vrf_id);
```

Comment:
do you need this, since we have the vrf id in the message already?

Reply:
fixed